docs/release_notes.md

```diff
@@ -1,5 +1,15 @@
 # Release Notes
 
+## Version 1.1.1
+
+### Bug Fixes
+
+- Fixed a bug in how the gravity was evaluated. Now the value from the UI represented in m/s<sup>2</sup> is converted to cm/s<sup>2</sup>.
```

Comment:
Should we include the adonis issue tag? Or as it was not shared with the users is not needed to keep track of it?

Reply:
I agree. It's added now.